app/api/merge/resume/route.ts

```
@@ -17,9 +22,13 @@ export async function GET(req: Request) {
   
   const attachmentId = url.searchParams.get('attachmentId');
 
-  // Check if attachmentId is provided and is a string
-  if (!attachmentId || typeof attachmentId !== 'string') {
-    return NextResponse.json({ error: 'Attachment ID is required and must be a string' }, { status: 400 });
+import { validateAttachmentId } from '@/app/api/merge/utils/security';
+
+  const attachmentId = url.searchParams.get('attachmentId');
```

Biome (1.9.4)

[error] Shouldn't redeclare 'attachmentId'. Consider to delete it or rename it.

'attachmentId' is defined here:


(lint/suspicious/noRedeclare)